lib/services/batch/package.json

```diff
@@ -21,7 +21,7 @@
     "ms-rest": "^2.2.2",
     "ms-rest-azure": "^2.3.3",
     "underscore": "^1.8.3",
-    "moment": "~2.18.1",
+    "moment": "^2.18.1",
```

Comment:
Please change this to "^2.21.0". This will make sure we will have a version higher than 2.18.1 (2.1.8.1 has security vulnerabilities ).

Reply:
@xingwu1 - Also update the minimum versions of ms-rest and ms-rest-azure to their latest versions ^2.3.2 and ^2.5.4 respectively.